server/server.js

```diff
@@ -2,31 +2,44 @@ require('newrelic');
 const express = require('express');
 const path = require('path');
 const mongoose = require('mongoose');
+const redis = require('redis');
+  redisClient = redis.createClient('redis://127.0.0.1:6379');
 const bodyParser = require('body-parser');
 const Product = require('../db/index.js');
 const cors = require('cors');
 const app = express();
+const responseTime = require('response-time');
+const port = 3001;
```

Comment:
You should organize this into absolute requires (i.e. npm modules) -> relative requires -> non-require statements like creating a redis client, rather than mixing them together. (Also, you didn't declare a variable when creating your client.)